demo/web/all_demos.dart

```diff
@@ -0,0 +1,404 @@
+/// [FirebaseElementDemo]
+/// [GoldCcCvcInputDemo]
+/// [GoldCcExpirationInputDemo]
+/// [GoldCcInputDemo]
+/// [GoldEmailInputDemo]
+/// [GoldPhoneInputDemo]
+/// [GoldZipInputDemo]
+/// [GoogleAnalyticsDemo]
+/// [GoogleApisDemo]
+/// [GoogleCalendarDemo]
+/// [GoogleCastableVideoDemo]
+/// [GoogleChartDemo]
+/// [GoogleDomainUserPickerDemo]
+/// [GoogleDriveDemo]
+/// [GoogleFeedsDemo]
+/// [GoogleHangoutButtonDemo]
+/// [GoogleMapDemo]
+/// [GoogleMapPolysDemo]
+/// [GoogleRecaptchaDemo]
+/// [GoogleSheetsDemo]
+/// [GoogleSigninDemo]
+/// [GoogleStreetviewPanoDemo]
+/// [GoogleUrlShortenerDemo]
+/// [GoogleYoutubeDemo]
+/// [GoogleYoutubeUploadDemo]
+/// [IronA11yAnnouncerDemo]
+/// [IronA11yKeysDemo]
+/// [IronA11yKeysBehaviorDemo]
+/// [IronAjaxDemo]
+/// [IronAutogrowTextareaDemo]
+/// [IronBehaviorsDemo]
+/// [IronCheckedElementBehaviorDemo]
+/// [IronCollapseDemo]
+/// [IronComponentPageDemo]
+/// [IronDocViewerDemo]
+/// [IronDropdownDemo]
+/// [IronFitBehaviorDemo]
+/// [IronFlexLayoutDemo]
+/// [IronFormDemo]
+/// [IronFormElementBehaviorDemo]
+/// [IronIconDemo]
+/// [IronIconsDemo]
+/// [IronIconsetDemo]
+/// [IronIconsetSvgDemo]
+/// [IronImageDemo]
+/// [IronInputDemo]
+/// [IronJsonpLibraryDemo]
+/// [IronLabelDemo]
+/// [IronListDemo]
+/// [IronListCollapseDemo]
+/// [IronListExternalContentDemo]
+/// [IronListSelectionDemo]
+/// [IronLocalstorageDemo]
+/// [IronMediaQueryDemo]
+/// [IronMenuBehaviorDemo]
+/// [IronMetaDemo]
+/// [IronOverlayBehaviorDemo]
+/// [IronPagesDemo]
+/// [IronRangeBehaviorDemo]
+/// [IronResizableBehaviorDemo]
+/// [IronSelectorDemo]
+/// [IronSignalsDemo]
+/// [IronValidatableBehaviorDemo]
+/// [IronValidatorBehaviorDemo]
+/// [MarkedElementDemo]
+/// [NeonAnimationCardDemo]
+/// [NeonAnimationDeclarativeDemo]
+/// [NeonAnimationDocDemo]
+/// [NeonAnimationDropdownDemo]
+/// [NeonAnimationGridDemo]
+/// [NeonAnimationListDemo]
+/// [NeonAnimationLoadDemo]
+/// [NeonAnimationReprojectionDemo]
+/// [NeonAnimationTilesDemo]
+/// [PaperBadgeDemo]
+/// [PaperBehaviorsDemo]
+/// [PaperButtonDemo]
+/// [PaperCardDemo]
+/// [PaperCheckboxDemo]
+/// [PaperDialogDemo]
+/// [PaperDialogBehaviorDemo]
+/// [PaperDialogScrollableDemo]
+/// [PaperDrawerPanelDemo]
+/// [PaperDropdownMenuDemo]
+/// [PaperFabDemo]
+/// [PaperHeaderPanelDemo]
+/// [PaperIconButtonDemo]
+/// [PaperInputDemo]
+/// [PaperItemDemo]
+/// [PaperListboxDemo]
+/// [PaperMaterialDemo]
+/// [PaperMenuDemo]
+/// [PaperMenuButtonDemo]
+/// [PaperProgressDemo]
+/// [PaperRadioButtonDemo]
+/// [PaperRadioGroupDemo]
+/// [PaperRippleDemo]
+/// [PaperScrollHeaderPanelDemo]
+/// [PaperSliderDemo]
+/// [PaperSpinnerDemo]
+/// [PaperStylesDemo]
+/// [PaperTabsDemo]
+/// [PaperToastDemo]
+/// [PaperToggleButtonDemo]
+/// [PaperToolbarDemo]
+/// [PaperTooltipDemo]
+@PolymerRegister('all-demos')
+class AllDemos extends PolymerElement {
+  AllDemos.created() : super.created();
+  List<DemoElementItem> __demos;
```

Comment:
This was a backing field for a getter/setter with a private setter, therefore the name `_emos` was already taken. Can't remember exactly why I did this. Just a leftover I forgot to remove.